lib/fog/aws/requests/compute/describe_images.rb

```diff
@@ -36,6 +36,8 @@ class Real
         #       * 'rootDeviceName'<~String> - Root device name, e.g. /dev/sda1
         #       * 'rootDeviceType'<~String> - Root device type, ebs or instance-store
         #       * 'virtualizationType'<~String> - Type of virtualization
+        #       * 'creationDate'<~Date> - Date the image was created
```

Comment:
`<~Date>` isn't actually true.  It is a `<~String>` until the parser comment is addressed.